Core/src/org/sleuthkit/autopsy/communications/Bundle.properties-MERGED

```diff
@@ -92,6 +92,7 @@ VisualizationPanel.forwardButton.text=
 VisualizationPanel.zoomPercentLabel.text=100%
 VisualizationPanel.zoomLabel.text=Zoom:
 VisualizationPanel.snapshotButton.toolTipText=Generate Snapshot report.
+SummaryViewer.contactsLabel.text=Entries in Book*:
```

Comment:
When I build this merged properties does not have the * between Book and the : and looks the same as the one in Bundle.properties I do not know if this means the MERGED file needs to be updated or it is one of our phantom diffs of merged files where they appear different for different users like the slashes.